tests/test_info_utils.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+
+import datasets.config
+from datasets.utils.info_utils import is_small_dataset
+
+
+@pytest.mark.parametrize("dataset_size", [None, 400 * 2 ** 20, 600 * 2 ** 20])
+@pytest.mark.parametrize("max_in_memory_dataset_size", ["default", None, 0, 100 * 2 ** 20, 900 * 2 ** 20])
+def test_is_small_dataset(dataset_size, max_in_memory_dataset_size):
```

Comment:
Maybe make sure that the `MAX_IN_MEMORY_DATASET_SIZE` returns to its original value after this test is run ?

To do so you can use a context manager like `unittest.mock.patch` for example

Reply:
Thanks, I had missed this...

I am using pytest monkeypatch instead. 😉